content/authorization/guides/system-vendor/system-user/byquery/_index.nb.md

```
@@ -0,0 +1,82 @@
+|---|---|---|---|
+| `system-id` | string | Ja | System-ID-en til systemet systembrukeren tilhører |
+| `orgno` | string | Ja | Organisasjonsnummeret til virksomheten som eier systembrukeren |
+| `external-ref` | string | Nei | Ekstern referanse satt ved opprettelse av systembrukeren |
+
+Parameteren `external-ref` er nyttig dersom det finnes flere systembrukere for samme kombinasjon av system og organisasjon.
+
+## Eksempel
+
+### Forespørsel
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)